src/components/Datepicker/utils.js

```diff
@@ -12,3 +12,681 @@ export const INTERNAL_VIEWS = {
   YEAR: 'YEAR',
   MONTH: 'MONTH',
 }
+
```

Comment:
Can we change this file to .ts too?

Reply:
The timezones can probably be moved to a higher utils file in the base of the design system to be used by other components too potentially